src/core/Elsa.Core/Options/ElsaOptions.cs

```diff
@@ -64,7 +64,7 @@ internal ElsaOptions()
         public Type DefaultWorkflowStorageProviderType { get; set; }
         public WorkflowChannelOptions WorkflowChannelOptions { get; set; } = new();
 
-        public bool UseTenantSignaler { get; set; } = false;
+        internal bool UseTenantSignaler { get; set; } = false;
```

Comment:
Although this makes sense, it also breaks anyone who's currently using this from their code. It's OK for them to have to use the new method, but what would you say of we keep this public for now but annotate it with an `ObsoleteAttribute` including a message that it is advised to use `UseTenantSignaler` instead?